README.md

```diff
@@ -74,8 +75,9 @@ android {
         javaCompileOptions {
             annotationProcessorOptions {
                 arguments = [
-                    stagGeneratedPackageName: 'com.vimeo.sample.stag.generated',
-                    stagDebug               : 'true'
+                    stagAssumeHungarianNotation: 'true'
+                    stagGeneratedPackageName   : 'com.vimeo.sample.stag.generated',
+                    stagDebug                  : 'true'
```

Comment:
I personally don't really like the use of annotation processor arguments in this instance. Up until now they're pretty much optional (and only necessary for the power user), but they'll be required for a person who uses Hungarian notation after this. 

I don't really know of a way to pass arguments to an AP other than this, but I know Gson solves the naming convention problem by having it as an argument to their builder. Stag is super lightweight and doesn't require a builder, and I'm not sure if the way Gson works for annotation processing is fundamentally different from stag. I guess I'm calling this out to see if there's a friendlier way to add it into the Stag API. 

The only two things I can think of are passing to a param to the StagFactory and not requiring the naming convention knowledge until Runtime (which I'm assuming is how Gson does it?) or by adding it as an argument at the class level `@UseStag` annotation which seems a little more appropriate but would be a pain to have to copy over to every annotation if you're using Hungarian.

Currently my two proposed solutions aren't better than the AP arg, but I'm just calling it out to see if you can think of anything.